test/component/scheduler/perf/util.go

```diff
@@ -109,54 +110,60 @@ func makeNodes(c client.Interface, nodeCount int) {
 	}
 }
 
-// makePods will setup specified number of scheduled pods.
-// Currently it goes through scheduling path and it's very slow to setup large number of pods.
-// TODO: Setup pods evenly on all nodes and quickly/non-linearly.
-func makePods(c client.Interface, podCount int) {
-	glog.Infof("making %d pods", podCount)
-	basePod := &api.Pod{
+func makePodSpec() api.PodSpec {
+	return api.PodSpec{
```

Comment:
It needs to set
```
Labels: map[string]string{"name": $RCName}
```
Because only in this way can we have [GetPodControllers()](https://github.com/kubernetes/kubernetes/blob/ebcc8f737c4ea7f21ba199657aedd03fb9a17845/plugin/pkg/scheduler/algorithm/priorities/selector_spreading.go#L92) return some selectors and non-zero selectors will lead to the matching of all pods.